utilities/setup/lint.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Simple alias script to install linting dependencies with HoloHub CLI:
+# holohub run-container --build-scripts lint
+# or:
+# holohub setup --scripts lint
+
+set -euxo pipefail
+
+holohub_command="holohub"
+if ! command -v ${holohub_command} &>/dev/null; then
+    if [[ -x ./holohub ]]; then
+        holohub_command="./holohub"
+    else
+        echo "Error: holohub not found in PATH or current directory." >&2
+        exit 1
+    fi
+fi
+
+
+${holohub_command} lint --install-dependencies
```

Comment:
I'm confused, now we have the CLI calling a container which calls a script which calls the CLI again? I thought your request to have separate scripts for installing dependencies (which I still think is unnecessary today) meant these scripts would be agnostic to holohub and just install dependencies explicitly.

Can you update this to avoid the cyclic dependency? So like:

`holohub lint --install-dependencies`
-> (if not `--local`, otherwise skip to next) build `lint.dockerfile`
-> run `lint.sh`
-> `pip install ...` or `apt install ...`

Reply:
more like what is done for coverage above